pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -251,6 +252,8 @@ self: super:
 
   libXvMC = super.libXvMC.overrideAttrs (attrs: {
     outputs = [ "out" "dev" "doc" ];
+    configureFlags = attrs.configureFlags or []
+      ++ malloc0ReturnsNullCrossFlag;
```

Comment:
@globin I assume your stuff covers this?

Reply:
Yes. The xserver fixes are already included.